### PR TITLE
test: 补充自适应在父容器有缩放的情况下宽高获取错误的测试

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

🔧 Chore

- [x] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description

补充 https://github.com/antvis/S2/pull/1425 缺失的单测

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔗 Related issue link

ref https://github.com/antvis/S2/pull/1425

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by lcx-seima — APPROVED** [review]

**Review by stone-lyl — APPROVED** [review]

File: packages/s2-react/__tests__/spreadsheet/adaptive-spec.tsx [hunk 1]
```diff
@@ -314,4 +315,44 @@ describe('SheetComponent adaptive Tests', () => {
...
+  test('should get original container size if container scaled', async () => {
+    const newContainerWidth = 1000;
+    const newContainerHeight = 500;
+    const containerId = 'scaleContainer';
+
+    act(() => {
+      ReactDOM.render(
+        <MainLayout
+          adaptive={{
+            getContainer: () => {
```
Comment (wjgogogo) [inline packages/s2-react/__tests__/spreadsheet/adaptive-spec.tsx:329]: 在render之间就将container中的`scale`改了会不会更符合实际情况一点

**Review by wjgogogo — APPROVED** [review]

**Review by xingwanying — DISMISSED** [review]
单测挂了

**Review by stone-lyl — APPROVED** [review]

**Pull request merged**